### ISSUE TITLE
[BGFIX] Failed to load scene.pkg file: Unsupported package version: PKGV0019

### ISSUE THREAD

**Issue by avidwriter** [Almamu/linux-wallpaperengine#185]
**Wallpaper Engine Background(s)**

https://steamcommunity.com/sharedfiles/filedetails/?id=3047352572

**Console output**

Failed to load scene.pkg file: Unsupported package version: PKGV0019

**Screenshots**

If you have any screenshot of it working on Windows that'll help so we can compare both outputs

**Desktop (please complete the following information):**
 
- OS: Ubuntu 23.04
 - Desktop Environment: GNOME
 - Window Manager: X11

**Additional context**

installed from git

**Comment by Iss-in** [comment]
same issue with multiple backgrounds, 3048130917, 3050040845

**Comment by trainzkid** [comment]
In `src/WallpaperEngine/Assets/CPackage.cpp`, during the header validation process (function `CPackage::validateHeader`, line 97), scene-style wallpapers are checked to see which "version" of ".pkg" file type (specific to scene-style wallpapers) they use. This app appears to only support wallpaper engine's ".pkg" file type versions 1 through 18. Anything else causes the aforementioned error. 

I'm not entirely sure what logic is tied to each version, but I'll keep digging (unless another contributor provides some more context here, as I'm just perusing through the cpp files to get some sort of idea, I'm no expert).

**TLDR: These scene-style wallpapers use a newer/different version of pkg file type structure that this app doesn't appear to support just yet**

**Comment by lovanto** [comment]
i got the same problem too

**Comment by Almamu** [comment]
I've changed the header validation to only care about PKGV and ignore the number. In theory the behavior should change between versions, but I haven't found any situation where it really matters yet. So this should be fixed now and for the future as long as the PKGV part is the same.
